c/meterpreter/source/metsrv/pool_party.h

```diff
@@ -0,0 +1,26 @@
+#include <windows.h>
+#define ERROR_POOLPARTY_VARIANT_FAILED					3
+#define ERROR_POOLPARTY_GENERIC							1
```

Comment:
If these get propagated back to Metasploit, I'm pretty sure they'll be treated as standard Win32 errors.

In that case `ERROR_POOLPARTY_GENERIC` would map to [ERROR_INVALID_FUNCTION](https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-), etc.

Reply:
They should not be propagated back to Metasploit, the `inject_via_poolparty` is setting a dwResult more suitable to metasploit if all variants fail.